src/AppBundle/DataFixtures/ORM/PostFixtures.php

```diff
@@ -91,11 +91,7 @@ public function getDependencies()
 
     private function addRandomTags(Post $post)
     {
-        if (0 === $count = mt_rand(0, 3)) {
```

Comment:
this means we don't have any post without tags in the fixtures. Is it expected ? Tagging posts is optional